src/UseCases/Addon/CreateAddonUseCase.java

```diff
@@ -30,7 +30,7 @@
      */
     public Boolean createAddon(String vendorToken, String name, float price,
                                ArrayList<Integer> types, boolean availability, String id) {
-        IVendor vendor = vendorRepository.getVendorFromToken(vendorToken);
+        IVendor vendor = (IVendor) vendorRepository.getUserFromToken(vendorToken);
```

Comment:
Why has this been renamed to getUserFromToken? Shouldn't this be getVendorFromToken?

Reply:
Vendor repository extends user repository so we have the getUserFromToken method

Reply:
right, makes sense